extensions/shared/block-icons.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import { isAtomicSite, isSimpleSite } from './site-type-utils';
+
+/**
+ * Returns the icon color for Jetpack blocks.
+ *
+ * Green in the Jetpack context, otherwise black for Simple sites or Atomic sites.
+ *
+ * @return {string} HEX color for block editor icons
+ */
+export function getIconColor() {
+	if ( isAtomicSite() || isSimpleSite() ) {
+		// Default Gutenberg G2 Black
+		// https://github.com/WordPress/gutenberg/blob/34ed2a6042d42fa18a5dcd0853d59bdff6a068d9/packages/base-styles/_colors.scss#L29
+		return '#1e1e1e';
+	}
+
+	// Jetpack Green
+	// https://github.com/Automattic/jetpack/blob/ed275e2d8167f79899c4965719f76410e244be87/_inc/client/scss/variables/_colors.scss#L95
+	return '#00be28';
```

Comment:
Does it matter this is not from the color studio?

Reply:
Good point, we should probably fix that. This happened a while ago, but I believe the colors were hardcoded for a specific reason — hence the comments.

Maybe we can do something like this, and call each value as strings:

https://github.com/Automattic/jetpack/blob/ea5269852da76e64683327b7e9fb976188cf4af1/modules/search/instant-search/components/jetpack-colophon.jsx#L8

Reply:
if it is done for a reason, that is OK... but the value itself might need to be updated? I think this is the old brighter green perhaps. I couldn't see it anyway.

Reply:
Well spotted. We need to use our Jetpack green :)